x11-misc/sxcs/sxcs-1.1.0.ebuild

```diff
@@ -0,0 +1,34 @@
+# Copyright 2021-2024 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit toolchain-funcs edo shell-completion
+
+DESCRIPTION="Color picker and magnifier for X11"
+HOMEPAGE="https://codeberg.org/NRK/sxcs"
+
+SRC_URI="https://codeberg.org/NRK/${PN}/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+S="${WORKDIR}/${PN}"
+
+KEYWORDS="~amd64"
+LICENSE="GPL-3+"
+SLOT="0"
+
+RDEPEND="
+	x11-libs/libX11
+	x11-libs/libXcursor
+"
+DEPEND="${RDEPEND}"
+
+src_compile() {
+	edo $(tc-getCC) -o sxcs sxcs.c ${CFLAGS} ${LDFLAGS} \
+	    -l X11 -l Xcursor || die "Compilation failed"
```

Comment:
No need to chain for here, edo will handle that for you

Reply:
Ah, right. Fixed.